choco/nunit-console-with-extensions.nuspec

```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nunit-console-with-extensions</id>
+    <version>0.0.0</version>
```

Comment:
This is how chocolatey generates templates, so it's at least their preferred approach. Don't know if they accept `$version$` as well.